reporter/src/main/kotlin/reporters/NoticeReporter.kt

```diff
@@ -38,7 +39,7 @@ class NoticeReporter : Reporter() {
 
     data class NoticeReport(
         val headers: List<String>,
-        val findings: LicenseFindingsMap,
+        val findings: Map<Identifier, Set<LicenseFindings>>,
```

Comment:
Commit message nit: You say "lists of `LicenseFindings`", but in the code it's actually sets.

Reply:
Done.